stable/gocd/values.yaml

```diff
@@ -200,7 +200,10 @@ agent:
     # agent.env.agentAutoRegisterResources is the GoCD Agent auto-register resources
     agentAutoRegisterResources:
     # agent.env.agentAutoRegisterEnvironments is the GoCD Agent auto-register Environments
-    agentAutoRegisterEnvironemnts:
+    # deprecated because of a typo. Use agent.env.agentAutoRegisterEnvironments instead
+    agentAutoRegisterEnvironemnts: env1, env2
```

Comment:
By setting this here, won't it end up going through the `else if` clause?

Reply:
Ah sorry, unintended checkin. Will fix it.